src/components/NewComment.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState, useContext, useEffect } from "react"
+import { UserContext } from "../contexts/User"
+import { postComment } from "../utils/apis"
+import ClipLoader from "react-spinners/ClipLoader";
+
+const NewComment = ({ setCommentWindowOpen, setComments, article_id, setCommentsExpanded }) => {
+
+    const { user, setUser } = useContext(UserContext)
+    const [newComment, setNewComment] = useState("")
+    const [postSuccess, setPostSuccess] = useState(false)
+    const [isError, setIsError] = useState(false)
+    const [isLoading, setIsLoading] = useState(false)
+
+    const handleSubmit = (e) => {
+        e.preventDefault()
+        setIsLoading(true)
+        postComment(newComment, article_id, user).then(postedComment => {
+            setComments(currentComments => {
+                return [postedComment, ...currentComments]
```

Comment:
Hi Matt, the second time you post a comment, everything breaks (though the API call goes through correctly) 
To save you some time debugging the root of the problem is that postedComments come back on the key of `comment`, whereas the currentComments aren't 🕵️ 🔍